content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Controllers
```

Comment:
> Also: Controller or Kontroler? I'd expect “Kontroler”.

I prefer Controller because "Kontroler" is weird, the official one should be "Pengontrol" but it sounds too general/common.